<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Tibor Auer</author>
 <class>rtQA</class>
 <widget class="QWidget" name="rtQA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>543</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>rtQA</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutControls">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl1">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lblStatus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedKingdom"/>
         </property>
         <property name="text">
          <string>IDLE</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Channels</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cbStimType">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Channel #1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Channel #2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dual-channel</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Current parameters</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl3_1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>  Sampling rate [Hz]</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="sbSamplingRate">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblFrequency1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>  Pulse f #1 [Hz]</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="sbFrequency1">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblFrequency2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>  Pulse f #1 [Hz]</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="sbFrequency2">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblPhase1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>  Phase #1 [deg]</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="sbPhase1">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblPhase2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>  Phase #2 [deg]</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="sbPhase2">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lblIntensity1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>  Amplitude #1 [mA]</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="sbIntensity1">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <double>9.990000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lblIntensity2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>  Amplitude #2 [mA]</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QDoubleSpinBox" name="sbIntensity2">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <double>9.990000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lbl4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Time parameters</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="lbl4_1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>  Duration [s]</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="sbDuration">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="lbl4_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>  Ramp up [s]</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="sbRampUp">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="lbl4_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>  Ramp down [s]</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QSpinBox" name="sbRampDown">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QPushButton" name="btnLoadConfig">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedKingdom"/>
         </property>
         <property name="text">
          <string>Load configuration...</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="16" column="0">
        <widget class="QPushButton" name="btnRun">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedKingdom"/>
         </property>
         <property name="text">
          <string>Run stimulation</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QPushButton" name="btnStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedKingdom"/>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QPushButton" name="btnLoadWaves">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedKingdom"/>
         </property>
         <property name="text">
          <string>Load waves</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutDualPlot">
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutCh1" stretch="0,1">
       <item>
        <widget class="QLabel" name="lblChannel1">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Channel #1</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="plChannel1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutCh2" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="lblChannel2">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Channel #2</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="plChannel2"/>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%p ms</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
